aviary/docs/getting_started/onboarding_level2.ipynb

```diff
@@ -775,9 +769,9 @@
     "```\n",
     "validation_cases/benchmark_tests/test_full_mission_solved_level3.py\n",
     "```\n",
-    "using `aircraft_for_bench_GwGm.csv` input deck. It is debatable whether it is an example in level 2 or level 3. We are treating it as level 2 because we use standard level 2 API, but as we will show in [level 3](onboarding_level3) onboarding document, `solved` missions are in geneal fall into level 3 category. Let us look at the key differences.\n",
+    "using the `aircraft_for_bench_GwGm_solved.csv` input deck. It can be run using level 2 or level 3. Here we are using the level 2 API, but as we will show in [level 3](onboarding_level3) onboarding document, `solved` missions are in geneal fall into level 3 category. Let us look at the key differences.\n",
```

Comment:
```suggestion
    "using the `aircraft_for_bench_GwGm_solved.csv` input deck. It can be run using level 2 or level 3. Here we are using the level 2 API, but as we will show in [level 3](onboarding_level3) onboarding document, `solved` missions generally fall into the level 3 category. Let us look at the key differences.\n",
```